docusaurus/docs/publish-a-plugin/sign-a-plugin.md

```diff
@@ -1,74 +1,88 @@
-| Commercial       | Yes;Commercial plugin subscription required | You can create, sign, and distribute plugins with dependent technologies that are closed source or commercially backed. To do so, enter into a Commercial plugin subscription with Grafana Labs.Commercial plugins are published on the Grafana plugin catalog, and are available to the entire Grafana community. |
+| **Plugin Level** | **Paid Subscription Required?**                 | **Description**                                                                                                                                                                                                                                                                                                                  |
+| ---------------- | ----------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Private          | No;<br></br>Free of charge                           | <p>You can create and sign a Private plugin for any technology at no charge.</p><p>Private plugins are intended for use on your own installation of Grafana. They may not be distributed to the Grafana community, and they are not published in the Grafana plugin catalog.</p>                                                 |
+| Community        | No;<br></br>Free of charge                           | <p>You can create, sign, and distribute plugins at no charge, provided that all dependent technologies are open source and not for profit.</p><p>Community plugins are published in the official Grafana catalog, and are available to the entire Grafana community.</p>                                                         |
+| Commercial       | Yes;<br></br>Commercial plugin subscription required | <p>You can create, sign, and distribute plugins with dependent technologies that are closed source or commercially backed. To do so, enter into a Commercial plugin subscription with Grafana Labs.</p><p>Commercial plugins are published on the Grafana plugin catalog, and are available to the entire Grafana community.</p> |
 
-For instructions on how to sign a plugin under the Community and Commercial signature level, refer to Sign a public plugin.
+For instructions on how to sign a plugin under the Community and Commercial signature level, refer to [Sign a public plugin](#sign-a-public-plugin).
```

Comment:
follow up, these links bring you back up to the previous section (will make a broader comment on this)

Reply:
Captured in https://github.com/grafana/plugin-tools/issues/360